### PR TITLE
add language selection

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review please

File: unicoremc/tests/base.py [hunk 1]
```diff
@@ -11,7 +12,7 @@
 from elasticgit.manager import StorageManager
 
 
-class UnicoremcTestCase(ModelBaseTest):
+class UnicoremcTestCase(TestCase, ModelBaseTest):
```
Comment (smn) [inline unicoremc/tests/base.py]: This seems a bit odd?

Comment (miltontony) [inline unicoremc/tests/base.py]: @smn yeah, it is a bit odd.. but I need `self.assertContains` which is provided by `django.tests.TestCase`

Comment (smn) [inline unicoremc/tests/base.py]: Judging by Django's [`TestCase`](https://github.com/django/django/blob/master/django/test/testcases.py#L903) I think what you want to include is [`TransactionTestCase`](https://github.com/django/django/blob/master/django/test/testcases.py#L745) instead.

**Comment by miltontony** [comment]
@smn ping on this

**Comment by smn** [comment]
:+1:

**Pull request merged**